doc/nrf/libraries/networking/lwm2m_location_assistance.rst

```diff
@@ -41,6 +41,74 @@ There are three different supported methods of obtaining the location assistance
 * Query of P-GPS predictions - The P-GPS predictions are queried from nRF Cloud and provided back to the device through the LwM2M server.
   The predictions are stored in the device flash and injected to the GNSS module when needed.
 
+API usage
+*********
+
+Cell based location
+===================
+
+Cell based location can be based only on current cell or current and neighboring cells.
+Getting the information about the current cell is achieved by using the Connectivity Monitor object (ID 4).
+By enabling the kconfig option :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_CONN_MON_OBJ_SUPPORT`, the connectivity monitor is populated with data about current cell.
+
+Getting the information about the neighboring cells is achieved by using a collection of ECID-Signal Measurement Information object (ID 10256).
+These are populated by calling :c:func:`lwm2m_ncell_handler_register` to register the listener for the neighborhood measurements and scheduling a measurement by calling :c:func:`lwm2m_ncell_schedule_measurement`.
+
+The Ground Fix Location object has only one resource that needs to be addressed before sending a location request, `report_back`.
+The back reporting tells the server if it needs to send the acquired location back to the device or not.
+If the location is sent back to the device, the location is stored only in the Ground Fix Location object.
+
+To finally send the location request for the cell based location, call :c:func:`location_assistance_ground_fix_request_send`.
+
+A-GPS location
+==============
+
+When using A-GPS location, the device requests A-GPS assistance data from the server.
+GNSS-module can be queried for the data it needs. Device can ask all the data in one go or split the request to reduce the memory usage.
```

Comment:
```suggestion
GPS module can be queried for the data it needs. The device can ask for all the data in one go or split the request to reduce the memory usage.
```

Reply:
Generally, I wouldn't talk about GNSS when this is GPS-only.

Reply:
The module is called GNSS module